console/src/test/java/org/web3j/console/project/ProjectTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 Web3 Labs LTD.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.console.project;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.web3j.TempFileProvider;
+
+public class ProjectTest extends TempFileProvider {
+    ProjectStructure projectStructure = new ProjectStructure(tempDirPath, "test", "test");
+    TemplateProvider templateProviderNew =
+            new TemplateProvider.Builder()
+                    .loadGradlewBatScript("gradlew.bat.template")
+                    .loadGradlewScript("gradlew.template")
+                    .loadMainJavaClass("Template.java")
+                    .loadGradleBuild("build.gradle.template")
+                    .loadGradleSettings("settings.gradle.template")
+                    .loadGradlewWrapperSettings("gradlew-wrapper.properties.template")
+                    .loadGradleJar("gradle-wrapper.jar")
+                    .loadSolidityGreeter("Greeter.sol")
+                    .withPackageNameReplacement(s -> s.replaceAll("<package_name>", "test"))
+                    .withProjectNameReplacement(s -> s.replaceAll("<project_name>", "test"))
+                    .build();
+
+    public ProjectTest() throws IOException {}
+
+    public void setUpProject() throws Exception {
+
+        Project project =
+                new Project.Builder()
+                        .withTemplateProvider(templateProviderNew)
+                        .withProjectStructure(projectStructure)
+                        .build();
+    }
+
+    @Test
+    public void directoryCreationTest() throws Exception {
+        setUpProject();
```

Comment:
please use an `@Before` annotated method similar to previous comments

Reply:
done